docs/howto/getting-started/enable-openstack-cli.md

```diff
@@ -0,0 +1,164 @@
+# Enabling the OpenStack CLI tool
```

Comment:
I think you can drop "tool" here.

Reply:
I sure can (and I just did).

Reply:
This is still unchanged in your branch; please update.